python/hail/methods/impex.py

```diff
@@ -1122,6 +1122,28 @@ def import_vcf(path, force=False, force_bgz=False, header_file=None, min_partiti
 @handle_py4j
 @typecheck(path=oneof(strlike, listof(strlike)))
 def index_bgen(path):
+    """Index .bgen files as required by :meth:`.methods.import_bgen`.
+
+    The index file is generated in the same directory as `path` with the
+    filename of `path` appended by `.idx`.
+
+    Example
+    -------
+
+    >>> hc.index_bgen("data/example3.bgen")
```

Comment:
Hail context is no longer exposed. `index_bgen` is in `impex`, shouldn't this be `methods.index_bgen`?